flocker/volume/test/filesystemtests.py

```diff
@@ -618,7 +620,11 @@ def got_volumes(copied):
                     with from_filesystem.reader() as reader:
                         with to_filesystem.writer() as writer:
                             data = reader.read()
-                            writer.write(data[1:])
+                            try:
+                                writer.write(data[1:])
+                            except IOError as e:
+                                if e.errno != errno.EPIPE:
```

Comment:
I have no idea what this code is doing. And I don't particularly care about it either, we're going to throw it away and replace the whole thing with something that isn't a horrible hack. So just do what you think best, it doesn't really matter.